src/LocalCache_TEST.cc

```diff
@@ -245,18 +245,20 @@ TEST_F(LocalCacheTest, IGN_UTILS_TEST_DISABLED_ON_WIN32(AllModels))
     uniqueNames.insert(iter->Identification().UniqueName());
     ++iter;
   }
+#ifndef _WIN32
   EXPECT_EQ(9u, uniqueNames.size());
 
   EXPECT_NE(uniqueNames.end(), uniqueNames.find(
       "http://localhost:8001/alice/models/am1"));
+#endif
```

Comment:
These are the main expectations of the test, what's the advantage of running it on Windows if these won't be checked? The same goes for the tests below.